student.rb

```diff
@@ -1,14 +1,17 @@
-require_relative 'person'
+require './person'
 
 class Student < Person
-  attr_accessor :classroom
-
```

Comment:
- [ ] please like the previous reviewer noted, kindly add getter method for @classroom here(in the student class).
`attr_reader :classroom`